Policies/github-terms-of-service.md

```diff
@@ -127,45 +128,58 @@ You agree not to engage in activity that significantly harms our Users. We will
 
-This license does not grant GitHub the right to sell your Content or otherwise distribute it outside of our Service.
+This license does not grant GitHub the right to sell Your Content or otherwise distribute or use it outside of our provision of the Service.
 
 #### 5. License Grant to Other Users
-Any Content you post publicly, including issues, comments, and contributions to other Users' repositories, may be viewed by others. By setting your repositories to be viewed publicly, you agree to allow others to view and "fork" your repositories (this means that others may make their own copies of your Content in repositories they control).
+Any User-Generated Content you post publicly, including issues, comments, and contributions to other Users' repositories, may be viewed by others. By setting your repositories to be viewed publicly, you agree to allow others to view and "fork" your repositories (this means that others may make their own copies of Content from your repositories in repositories they control).
 
-If you set your pages and repositories to be viewed publicly, you grant each User of GitHub a nonexclusive, worldwide license to access your Content through the GitHub Service, and to use, display and perform your Content, and to reproduce your Content solely on GitHub as permitted through GitHub's functionality. You may grant further rights if you [adopt a license](/articles/adding-a-license-to-a-repository/#including-an-open-source-license-in-your-repository).
+If you set your pages and repositories to be viewed publicly, you grant each User of GitHub a nonexclusive, worldwide license to use, display, and perform Your Content through the GitHub Service and to reproduce Your Content solely on GitHub as permitted through GitHub's functionality (for example, through forking). You may grant further rights if you [adopt a license](/articles/adding-a-license-to-a-repository/#including-an-open-source-license-in-your-repository). If you are uploading Content you did not create or own, you are responsible for ensuring that the Content you upload is licensed under terms that grant these permissions to other GitHub Users.
 
 #### 6. Contributions Under Repository License
-Whenever you make a contribution to a repository containing notice of a license, you license your contribution under the same terms, and you agree that you have the right to license your contribution under those terms. If you have a separate agreement to license your contributions under different terms, such as a contributor license agreement, that agreement will supercede.
+Whenever you make a contribution to a repository containing notice of a license, you license your contribution under the same terms, and you agree that you have the right to license your contribution under those terms. If you have a separate agreement to license your contributions under different terms, such as a contributor license agreement, that agreement will supersede.
```

Comment:
@gs-rezaem @nsqe I mean it in a good way, in that it is written in a way that covers situations that aren't immediately obvious without having to explicitly list or state them.

As written, a specific agreement always takes precedence over the generic licence without completely removing it.  That means that when there is a explicit licencing agreement, such as in a CLA or contribution rules, but someone HASN'T explicitly agreed to it, there's no period in which their contributions aren't covered by the licence.

I'll admit that I can't think exactly when this might be useful in terms of PRs, but it's a good thing to have just in case, especially if the standing licence is something with a fairly permissive grant.  
What it does mean is that there aren't cases where there are contributions in issue and PRs, even just in terms of actual issue text, that aren't actually under an open source licence at all, everything added to the project in any way is automatically covered without the project owners having to do anything beyond specifying a licence.

Does it make it sound more useful in that framing? :)